dotnetv3/EC2/README.md

```diff
@@ -13,19 +13,41 @@ Amazon EC2 is a web service that provides resizable computing capacity—literal
 
 ## Code examples
 
+### Getting started
+* [Hello Amazon EC2](actions/HelloEc2.cs)
+
 ### Single actions
 Code excerpts that show you how to call individual service functions.
-* [Create a security group](CreateSecurityGroupExample/CreateSecurityGroupExample/CreateSecurityGroup.cs) (`CreateSecurityGroupAsync`)
-* [Create a security key pair](CreateKeyPairExample/CreateKeyPairExample/CreateKeyPair.cs) (`CreateKeyPairAsync`)
-* [Create a VPC](VirtualPrivateCloudExamples/CreateVPCExample/CreateVPC.cs) (`CreateVpcAsync`)
+* [AllocateAddress](actions/EC2Wrapper.cs) (`AllocateAddressAsync`)
+* [AssociateAddress](actions/EC2Wrapper.cs) (`AssociateAddressAsync`)
+* [AuthorizeSecurityGroupIngress](actions/EC2Wrapper.cs) (`AuthorizeSecurityGroupIngressAsync`)
+* [Create a security group](actions/EC2Wrapper.cs) (`CreateSecurityGroupAsync`)
+* [Create a security key pair](actions/EC2Wrapper.cs) (`CreateKeyPairAsync`)
+* [Create a VPC](actions/EC2Wrapper.cs) (`CreateVpcAsync`)
 * [Create a VPC endpoint to use with an Amazon Simple Storage Service (Amazon S3) client.](VirtualPrivateCloudExamples/CreateVPCforS3Example/CreateVPCforS3.cs) (`CreateVpcAsync`)
-* [Delete a security group](DeleteSecurityGroupExample/DeleteSecurityGroupExample/DeleteSecurityGroup.cs) (`DeleteSecurityGroupAsync`)
-* [Delete a security key pair](DeleteKeyPairExample/DeleteKeyPairExample/DeleteKeyPair.cs) (`DeleteKeyPairAsync`)
-* [Delete a VPC](DeleteVPCExample/DeleteVPCExample/DeleteVPC.cs) (`DeleteVpcAsync`)
-* [Describe instances](DescribeInstancesExample/DescribeInstancesExample/DescribeInstances.cs) (`DescribeInstancesAsync`)
-* [Reboot an instance](RebootInstancesExample/RebootInstancesExample/RebootInstances.cs) (`RebootInstancesAsync`)
-* [Start an instance](StartInstancesExample/StartInstancesExample/StartInstances.cs) (`StartInstancesAsync`)
-* [Stop an instance](StopInstancesExample/StopInstancesExample/StopInstances.cs) (`StopInstancesAsync`)
+* [Delete a security group](actions/EC2Wrapper.cs) (`DeleteSecurityGroupAsync`)
+* [Delete a security key pair](actions/EC2Wrapper.cs) (`DeleteKeyPairAsync`)
+* [Delete a VPC](actions/EC2Wrapper.cs) (`DeleteVpcAsync`)
+* [DescribeImages](actions/EC2Wrapper.cs) (`DescribeImagesAsync`)
+* [Describe instances](actions/EC2Wrapper.cs) (`DescribeInstancesAsync`)
+* [DescribeInstances](actions/EC2Wrapper.cs) (`DescribeInstancesAsync`)
```

Comment:
Same here as above. This casing is inconsistent with the others.

Reply:
Edited the descriptions according to the examples in the code examples library.